docs/source/models.rst

```diff
@@ -2317,6 +2317,29 @@ dataframe's column names must match the model signature's column names.
     pyfunc_udf = mlflow.pyfunc.spark_udf(spark, <path-to-model-with-signature>)
     df = spark_df.withColumn("prediction", pyfunc_udf())
 
+If a model contains a signature with tensor spec inputs,
+you need to pass a column of array type as a corresponding UDF argument,
+the column values must be one dimension arrays and the
+UDF will reshape the column values to the required shape with C_CONTIGUOUS (row-major)
+order and cast the values as the required tensor spec type. For example, assuming a model
+requires input 'a' of shape (-1, 2, 3) and input 'b' of shape (-1, 4, 5), then we need to
+prepare an inference spark dataframe with column 'a' containing arrays of length 6 and
+column 'b' containing arrays of length 20, then invoke the UDF like following example code.
+
+.. rubric:: Example
+
+.. code-block:: py
```

Comment:
should this be `test-code-block:: py`?

Reply:
Other example code in this file also uses `code-block:: py`